packages/react-components/react-storybook-addon/src/components/DirSwitch.styles.ts

```diff
@@ -0,0 +1,7 @@
+import { makeStyles } from '@griffel/react';
```

Comment:
not sure this was the best architectural decision , for storybook UIs we should stick with storybook styling approach